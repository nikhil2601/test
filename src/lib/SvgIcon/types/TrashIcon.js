import React from 'react';
import PropTypes from 'prop-types';

import SvgIcon from '../SvgIcon';

const TrashIcon = ({ setRef, ...rest }) => (
    <SvgIcon {...rest} ref={setRef}>
        <path d="M19,4H15.5L14.5,3H9.5L8.5,4H5V6H19M6,19A2,2 0 0,0 8,21H16A2,2 0 0,0 18,19V7H6V19Z" />
    </SvgIcon>
);

TrashIcon.propTypes = {
    setRef: PropTypes.object,
    viewBox: PropTypes.string,
};

TrashIcon.defaultProps = {
    setRef: null,
    viewBox: '0 0 24 24',
};

export default TrashIcon;
