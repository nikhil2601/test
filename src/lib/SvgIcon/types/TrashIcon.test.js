import React from 'react';
import { shallow } from 'enzyme';

import TrashIcon from './TrashIcon';

describe('<TrashIcon />', () => {
    const wrapper = shallow(<TrashIcon />);

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
