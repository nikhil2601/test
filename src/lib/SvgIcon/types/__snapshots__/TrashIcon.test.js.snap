// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrashIcon /> should render correctly 1`] = `
<SvgIcon
  color="inherit"
  height="1rem"
  margin={null}
  opacity={null}
  padding={null}
  titleAccess={null}
  viewBox="0 0 24 24"
  width="1rem"
>
  <path
    d="M19,4H15.5L14.5,3H9.5L8.5,4H5V6H19M6,19A2,2 0 0,0 8,21H16A2,2 0 0,0 18,19V7H6V19Z"
  />
</SvgIcon>
`;
